ar_exercise/exercise.rb

```diff
@@ -31,41 +34,91 @@ class Apartment < ActiveRecord::Base
 
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
+# Before running pry, run file through ruby (fil)
+
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+all_tenants = Tenant.first
+
 # get all tenants older than 65
+all_tenants = Tenant.where("age: > ?" "65")
+
 # get all apartments whose price is greater than $2300
+all_apartments = Apartment.where("age: > ?" "2300")
+
 # get the apartment with the address "6005 Damien Corners"
+all_apartments = Apartment.where(address:"6005 Damien Corners")
+
 # get all tenants in that apartment
+all_tenants = Tenant.where(apartment_id: 6)
```

Comment:
Once you have the apartment above in a variable, a better way to get the tenants would be:

```
tenants = apartment_at_damien_corners.tenants
```